### DISCUSSION TITLE
SKFilterQuality obsolete?

### DISCUSSION THREAD

**Discussion by praxmarer** [mono/SkiaSharp#3116] [Q&A]
Hi,

i use this example here : https://github.com/mono/SkiaSharp/issues/836#issuecomment-584895517

But in the current version FilterQuality is obsolete - i already looked through release notes - but i cannot find what would be the replacement - has anybody any tipps?

 //paint.FilterQuality = SKFilterQuality.High;

**Comment by Pangamma** [comment]
Did you find any answer?

**Comment by noxe** [comment]
No

**Comment by TobiasW-T** [comment]
The answer is shown here: https://github.com/mono/SkiaSharp/discussions/3163#discussioncomment-12182634

**Comment by menmories** [comment]
using SKSamplingOptions,for example:
SKRect rcImage = new SKRect(0, 0, bitmap.Width, bitmap.Height);
SKRect rcTarget = new SKRect(0, 0, ClientSize.Width, ClientSize.Height);
var sampling = new SKSamplingOptions(SKFilterMode.Linear, SKMipmapMode.Linear);
var resizedBitmap = bitmap.Resize(new SKImageInfo((int)rcTarget.Width, (int)rcTarget.Height), sampling);
canvas.DrawBitmap(bitmap, rcTarget, paint);
